pallets/gear-program/src/lib.rs

```diff
@@ -39,11 +42,15 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+pub mod migration;
 pub mod weights;
 
 #[cfg(feature = "runtime-benchmarks")]
 mod benchmarking;
 
+/// The current storage version.
+const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Isn't it better to have a different name for this constant in every pallet to not be confused when incrementing?
```suggestion
const PROGRAM_STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```